README.md

```diff
@@ -10,37 +10,53 @@ This package presently imports fluid dynamics simulation data and plots the best
 * config.model.yml has all of the user-set values for the model functions is called by the main function.
 * RocketRL.python.main.py is the main function that calls the other scripts and data.
 * RocketRL.python.func.data_processing.py has all of the data importing, processing, and visualization functions.
-* RocketRL.python.func.run_env.py runs the agent in the custom environment.
-* RocketRL.python.envs.my_collection.my_awesome_env.py contains the custom env for this task.
+* RocketRL.python.func.ray_funcs.py plots some of the outputs from the RL algorithm 
+* RocketRL.python.envs.RocketRL.RocketEnv_2T.py contains the custom env for this task.
 
 ## To run:
+
+(note to self: update req.txt)
 1. This only works for OSX: Install requirements from req.txt; if you're using conda, create your new environment as below: 
 
-```Terminal
+```bash
 $ conda create -n newenvironment --file req.txt
 ```
+2. Install Ray: 
+- A few notes: Download the latest tag instead of the main branch b/c main branch doesn't build
+```bash
+git clone https://github.com/ray-project/ray.git
+cd ray
+git checkout tags/ray-0.4.0
+```
+- (Install instructions)[http://ray.readthedocs.io/en/latest/installation.html] 
+
+3. Install RLlib:
+- install snappy first if you don't already have it:
+```bash
+$ brew install snappy
+```
+-(Install instructions)[https://ray.readthedocs.io/en/latest/rllib.html]
 
-2. In the root of your github repo, type :
+4. In the root of your github repo, type :
 ```Bash
 export PYTHONPATH=$PYTHONPATH:`pwd`/RocketRL/python
 ```
 
-3. Copy files that are in the \envs folder into your gym/envs folder (with algorithmic, classic_control, etc. folders in it). The lines in \envs\__init__.py that were added to the pre-existing file are: 
-
-```python 
-from gym.envs.my_collection.my_awesome_env import SimpleCorridor
-register(
-   	id='SimpleCorridor-v0',
-   	entry_point='gym.envs.my_collection:SimpleCorridor',
-)
-```
+5. The contents of the env folder have to be copied to openAI gym envs folder for the rollout to work properly:
```

Comment:
Even though I can register and run the environment to train the model, the rollout won't find this environment since it only looks in the gym/envs folder. Maybe I can write a bash function that moves these files into that folder?